### PR TITLE
[PIN-1677] Adding limitations to strings fields inside yml

### PR DESCRIPTION
**Pull request by bloodsky**

### CODE REVIEW

**Review by TonioGela — APPROVED** [review]
Thanks a lot, it should have been quite tedious

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -612,6 +612,7 @@ paths:
                     - DOCUMENT
                 prettyName:
                   type: string
+                  maxLength: 60
```
Comment (galales) [inline src/main/resources/interface-specification.yml:616]:
```suggestion
                  minLength: 5
                  maxLength: 60
```

File: src/main/resources/interface-specification.yml [hunk 2]
```diff
@@ -825,8 +826,10 @@ components:
           format: uuid
         name:
           type: string
+          maxLength: 60
```
Comment (galales) [inline src/main/resources/interface-specification.yml:827]:
```suggestion
          minLength: 5
          maxLength: 60
```

File: src/main/resources/interface-specification.yml [hunk 3]
```diff
@@ -825,8 +826,10 @@ components:
...
         description:
           type: string
+          maxLength: 250
```
Comment (galales) [inline src/main/resources/interface-specification.yml:831]:
```suggestion
          minLength: 10
          maxLength: 250
```

File: src/main/resources/interface-specification.yml [hunk 4]
```diff
@@ -841,8 +844,10 @@ components:
       properties:
         name:
           type: string
+          maxLength: 60
```
Comment (galales) [inline src/main/resources/interface-specification.yml:847]:
```suggestion
          minLength: 5
          maxLength: 60
```

File: src/main/resources/interface-specification.yml [hunk 5]
```diff
@@ -841,8 +844,10 @@ components:
...
         description:
           type: string
+          maxLength: 250
```
Comment (galales) [inline src/main/resources/interface-specification.yml:851]:
```suggestion
          minLength: 10
          maxLength: 250
```

File: src/main/resources/interface-specification.yml [hunk 6]
```diff
@@ -858,10 +863,12 @@ components:
       properties:
         description:
           type: string
+          maxLength: 250
         audience:
           type: array
           items:
             type: string
+            maxLength: 250
```
Comment (galales) [inline src/main/resources/interface-specification.yml:874]:
```suggestion
            minLength: 1
            maxLength: 250
```

File: src/main/resources/interface-specification.yml [hunk 7]
```diff
@@ -890,10 +897,12 @@ components:
...
```
Comment (galales) [inline src/main/resources/interface-specification.yml:910]:
```suggestion
            minLength: 1
            maxLength: 250
```

File: src/main/resources/interface-specification.yml [hunk 8]
```diff
@@ -918,6 +927,7 @@ components:
       properties:
         prettyName:
           type: string
+          maxLength: 60
```
Comment (galales) [inline src/main/resources/interface-specification.yml:936]:
```suggestion
          minLength: 5
          maxLength: 60
```

File: src/main/resources/interface-specification.yml [hunk 9]
```diff
@@ -994,8 +1004,10 @@ components:
           $ref: '#/components/schemas/Organization'
         name:
           type: string
+          maxLength: 60
```
Comment (galales) [inline src/main/resources/interface-specification.yml]: I suppose constraints on output payloads are just informative?
as they are not actually used for any check

**Review by galales — COMMENTED** [review]
nice work, thank you.
I just suggested a `minLength` property for fields that are mandatory, to avoid the caller to provide an empty string.
Values are just an educated guess.

Have constraints on output-only payloads been added with a specific purpose?

File: src/main/resources/interface-specification.yml (hunk 9 above)
Comment (bloodsky) [reply src/main/resources/interface-specification.yml]: I made a mistake with outputs payloads. I'm removing constraints on them

**Review by galales — APPROVED** [review]

**Pull request merged**